lorax.spec

```diff
@@ -224,10 +224,10 @@ getent passwd weldr >/dev/null 2>&1 || useradd -r -g weldr -d / -s /sbin/nologin
 %dir %attr(0771, root, weldr) %{_sharedstatedir}/lorax/composer/blueprints/
 %attr(0771, weldr, weldr) %{_sharedstatedir}/lorax/composer/blueprints/*
 
-%files -n composer-cli
-%{_bindir}/composer-cli
+%files -n composer
+%{_bindir}/composer
```

Comment:
I think we need to ship a symlink to cover the old name. What do you think? I think we may need to do that on RHEL.